src/dms-preview/azext_dms/custom.py

```diff
@@ -438,11 +454,10 @@ def run_command(client,
 
 
 def get_scenario_type(source_platform, target_platform, task_type=""):
-    if source_platform == "sql":
-        if target_platform == "sqldb":
-            scenario_type = ScenarioType.sql_sqldb_online if "online" in task_type else ScenarioType.sql_sqldb_offline
-        elif target_platform == "sqlmi":
-            scenario_type = ScenarioType.sql_sqlmi_online if "online" in task_type else ScenarioType.sql_sqlmi_offline
+    if source_platform == "sql" and target_platform == "sqldb":
```

Comment:
Why this change? They are functionally identical and I split it out so that if there was branching based on target it would be easier to read. If you feel strongly the change makes it easier to read I shan't push back.

Reply:
Adressed in the comment above.

Reply:
So it forces it to fall through to type Unknown?

Reply:
Sorry, I thought this was referring to ln 148. 

For this change, if the source is 'sql' but the target doesn't match "sqlmi" or "sqldb", it doesn't end up setting scenario_type.